src/encoded/static/components/navigation.js

```diff
@@ -205,9 +224,9 @@ Navigation.childContextTypes = {
     dropdownClick: PropTypes.func, // Called when a dropdown title gets clicked
 };
 
-
 // Main navigation menus
 const GlobalSections = (props, context) => {
+    const glossary = require('./glossary/glossary.json');
```

Comment:
I didn’t know you could do this. This is basically how I imagined it if the JSON was in a .js file, but in a JSON file. This embeds the JSON in the JS as an object, which I think is a good way to do this.

Reply:
It's very simple!